CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Updated
```

Comment:
I would call this header either `Changed` or `Fixed`. There's no `Updated` section defined at https://keepachangelog.com/en/1.0.0/#how.

I would call it `Changed` if there's any chance at all something could be depending on the original directory structure, and would make that a major version bump.

Otherwise I would call it `Fixed`.

Reply:
Good point. Will update to "Changed". All the deployments should be using the tasks from npm so I don't expect anyone relying the folder structure.

Reply:
It should be listed under `Fixed`. This is only important because if there's a `Changed` section under `Unreleased` then that's an indicator that the next release needs to be a major version bump.